expression/builtin_math_vec.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The ql Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2019 The ql Authors. All rights reserved.
```

Reply:
Unnecessary to state `ql` license here?

Reply:
Yes, it's my mistake.
It's updated now.